packages/ag-charts-community/src/api/preset/sparkline.ts

```diff
@@ -67,6 +68,26 @@ const crossHairTooltip = {
     },
 };
 
+const barGridLineDefaults: AgAxisGridLineOptions = {
+    style: [{ stroke: DEFAULT_AXIS_GRID_COLOUR }],
+    width: 2,
+};
+
+const barAxisDefaults: AgCommonThemeableAxisOptions = {
+    number: {
+        gridLine: barGridLineDefaults,
+    },
+    log: {
+        gridLine: barGridLineDefaults,
+    },
+    time: {
+        gridLine: barGridLineDefaults,
+    },
+    category: {
+        gridLine: barGridLineDefaults,
+    },
```

Comment:
Do we need ordinal time too?

Reply:
Actually, current spec is only category, number and time. I'll remove log.